spec/lib/traject/macros/custom_spec.rb

```diff
@@ -263,4 +263,15 @@
       end
     end
   end
+
+  describe '#exclude_locations' do
+    let(:location_1) { { '949' => { 'ind1' => '', 'ind2' => '', 'subfields' => [{ 'l' => 'DOCUSMF-DN' }] } } }
+    let(:location_2) { { '949' => { 'ind1' => '', 'ind2' => '', 'subfields' => [{ 'l' => 'RESERVE-HN' }] } } }
+    let(:location_3) { { '949' => { 'ind1' => '', 'ind2' => '', 'subfields' => [{ 'l' => 'AVAIL_SOON' }] } } }
+    let(:result) { @indexer.map_record(MARC::Record.new_from_hash('fields' => [location_1, location_2, location_3], 'leader' => leader)) }
```

Comment:
latest version of rubocop seem to wanting to use let memoization less. 🤷‍♂️ It does feel easier to read and it is shorter.

```suggestion
    location_1 = { '949' => { 'ind1' => '', 'ind2' => '', 'subfields' => [{ 'l' => 'DOCUSMF-DN' }] } }
    location_2 = { '949' => { 'ind1' => '', 'ind2' => '', 'subfields' => [{ 'l' => 'RESERVE-HN' }] } }
    location_3 = { '949' => { 'ind1' => '', 'ind2' => '', 'subfields' => [{ 'l' => 'AVAIL_SOON' }] } }
    result = @indexer.map_record(MARC::Record.new_from_hash('fields' => [location_1, location_2, location_3], 'leader' => leader))
```